### ISSUE TITLE
Thimble dont´start

### ISSUE THREAD

**Issue by logoht1** [mozilla/thimble.mozilla.org#2672]
Thimble could not be started.
Please reload your browser to reload Thimble.
Details of this error can be found in the console of the developer tools of your browser.
NOTE: If using private mode, please reload in normal mode.
The standard modus not work either. 
What can I do. 
Thx

**Comment by Pomax** [comment]
I am seeing thimble start normally for me in Chrome and Firefox, so the first thing is probably to figure out the situation:
- which OS are you using?
- which browser are you using?
- does that browser have addons/extensions that interfere with cookies and/org localstorage? (privacy, ad-blocking, etc)
- when you open the developer tools for that browser, does its "console" show any errors when you try to load thimble?

**Comment by logoht1** [comment]
Thimble start normally for me in Chrome and Safari. In Firefox unfortunately not.
I have version Quantum 62.0(64 bit). OS-version is High Sierra 10.13.1. 
All plugins and addons are disabled. The console shows following:
Content Security Policy: Die Direktive 'child-src' sollte nicht mehr verwendet werden. Bitte verwenden Sie stattdessen die Direktive 'worker-src' zum Kontrollieren von Workern bzw. die Direktive 'frame-src' zum Kontrollieren von Frames.
[Thimble] to see detailed logging info in the console, reload with ?logging=1 on the URL. main.js:24:77194
Filer error:  TypeError: "this.db is null"
_getObjectStorehttps://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:11241_gethttps://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:11548getObjecthttps://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:11744shttps://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:2:18680ohttps://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:5315onerrorhttps://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:12779 bramble.js:3:3694
[Bramble] Failed to load Project state, with 
Object { name: "EFILESYSTEMERROR", code: "EFILESYSTEMERROR", errno: 1001, message: "unknown error", stack: "e@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:2:16074\no/this.queueOrRun@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:4597\n[20]</</o.prototype[t]@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:6245\na@https://thimble.mozilla.org/editor/scripts/main.js:29:10378\nl/<@https://thimble.mozilla.org/editor/scripts/main.js:29:10591\ni@https://thimble.mozilla.org/editor/scripts/main.js:29:10253\nl@https://thimble.mozilla.org/editor/scripts/main.js:29:10578\nO@https://thimble.mozilla.org/editor/scripts/main.js:24:72144\ni@https://thimble.mozilla.org/editor/scripts/main.js:24:92998\ns@https://thimble.mozilla.org/editor/scripts/main.js:24:80473\n@https://thimble.mozilla.org/editor/scripts/main.js:24:93253\nu@https://thimble.mozilla.org/editor/scripts/main.js:24:25133\no/</c<@https://thimble.mozilla.org/editor/scripts/main.js:24:25449\n" }
main.js:24:93025
[Thimble Error] 
Object { name: "EFILESYSTEMERROR", code: "EFILESYSTEMERROR", errno: 1001, message: "unknown error", stack: "e@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:2:16074\no/this.queueOrRun@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:4597\n[20]</</o.prototype[t]@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:6245\na@https://thimble.mozilla.org/editor/scripts/main.js:29:10378\nh/<@https://thimble.mozilla.org/editor/scripts/main.js:29:11310\ni@https://thimble.mozilla.org/editor/scripts/main.js:29:10253\nh@https://thimble.mozilla.org/editor/scripts/main.js:29:11297\nE@https://thimble.mozilla.org/editor/scripts/main.js:24:71608\nN@https://thimble.mozilla.org/editor/scripts/main.js:24:73096\ni@https://thimble.mozilla.org/editor/scripts/main.js:29:12961\ni/<@https://thimble.mozilla.org/editor/scripts/main.js:24:93149\nO/<@https://thimble.mozilla.org/editor/scripts/main.js:24:72206\no/<@https://thimble.mozilla.org/editor/scripts/main.js:29:10337\nl/</<@https://thimble.mozilla.org/editor/scripts/main.js:29:10621\na/<@https://thimble.mozilla.org/editor/scripts/main.js:29:10439\n[20]</</o.prototype[t]@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:6533\na@https://thimble.mozilla.org/editor/scripts/main.js:29:10378\nl/<@https://thimble.mozilla.org/editor/scripts/main.js:29:10591\ni@https://thimble.mozilla.org/editor/scripts/main.js:29:10253\nl@https://thimble.mozilla.org/editor/scripts/main.js:29:10578\nO@https://thimble.mozilla.org/editor/scripts/main.js:24:72144\ni@https://thimble.mozilla.org/editor/scripts/main.js:24:92998\ns@https://thimble.mozilla.org/editor/scripts/main.js:24:80473\n@https://thimble.mozilla.org/editor/scripts/main.js:24:93253\nu@https://thimble.mozilla.org/editor/scripts/main.js:24:25133\no/</c<@https://thimble.mozilla.org/editor/scripts/main.js:24:25449\n" }
main.js:29:12996
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: call to eval() or related function blocked by CSP. 5f0582c6-00c1-744f-a7f1-102240530f11:27
Die Seite wurde neu geladen, weil die Zeichenkodierungs-Deklaration des HTML-Dokuments im Frame beim Vorverarbeiten der ersten 1024 Zeichen der Datei nicht gefunden wurde. Die Kodierungs-Deklaration muss in die ersten 1024 Zeichen der Datei verschoben werden. index.html:27
Laden fehlgeschlagen für das <script> mit der Quelle "blob:https://mozillathimblelivepreview.net/f9530cbe-331a-6c45-9e47-3bf80f398480". index.html:1
[Bramble] browser filesystem inaccessible (needs to be re-formatted, or permission denied due to private browsing mode. bramble.js:4:883
[Bramble Error] error loading Bramble 
Object { name: "EFILESYSTEMERROR", code: "EFILESYSTEMERROR", errno: 1001, message: "unknown error", stack: "e@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:2:16074\no/this.queueOrRun@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:4597\n[20]</</o.prototype[t]@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:3:6245\n_@https://mozillathimblelivepreview.net/bramble/production/dist/bramble.js:4:6674\n" }
main.js:24:79326
[Bramble] browser filesystem inaccessible (needs to be re-formatted, or permission denied due to private browsing mode. bramble.js:4:883


















































































Am 17.09.2018 um 22:44 schrieb Pomax <notifications@github.com>:

**Comment by mischkl** [comment]
Safari and Chrome on Mac OS, Thimble shows the loading screen but never loads :(

**Comment by Pomax** [comment]
@mischkl unless you have the exact same problem (with the same error log), please file that as a new issue, because it may be for completely different reasons.

@logoht1 it looks like that `TypeError: "this.db is null"` is the main culprit, so you may need to manually help firefox reset its indexdb so you can use it with thimble.  https://github.com/mozilla/thimble.mozilla.org/wiki/Reset-indexedDB explains how you can go about doing this.

**Comment by mischkl** [comment]
@Pomax it's working now of its own accord, thanks.

**Comment by Pomax** [comment]
@logoht1 it looks like that `TypeError: "this.db is null"` is the main culprit, so you may need to manually help firefox reset its indexdb so you can use it with thimble.  https://github.com/mozilla/thimble.mozilla.org/wiki/Reset-indexedDB explains how you can go about doing this.
